extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -62,10 +69,11 @@ public function executeShell() {
 		global $IP;
 
 		$cmd = sprintf(
-			"SERVER_ID=%d %s %s/maintenance/importStarter.php",
+			"SERVER_ID=%d %s %s/maintenance/importStarter.php --vertical=%d",
 			$this->taskContext->getCityId(),
 			$this->phpBin,
-			"{$IP}/extensions/wikia/CreateNewWiki"
+			"{$IP}/extensions/wikia/CreateNewWiki",
+			$this->taskContext->getVertical()
```

Comment:
I talked about it with @jcellary and he told me this is risky. There are multiple race conditions in the process already and I don't want to add a new one. I think that passing an option explicitly is better than depending on a global state which might be outdated (slave lag etc.).